kotlin/README.md

```diff
@@ -0,0 +1,38 @@
+# truelayer-signing
+Kotlin/Java package to produce & verify TrueLayer API requests signatures.
+
+### Kotlin usage
+```kotlin
+// `Tl-Signature` value to send with the request.
+Signer.from(KID, privateKey)
+    .header("Idempotency-Key", idempotencyKey)
+    .method("post")
+    .path(path)
+    .body(body)
+    .sign()
+```
+
+### Java usage
```

Comment:
We'll probably want Java at the top, as it's most likely what clients will be after